### PR TITLE
Deploy to all staging/production helplines

### PR DESCRIPTION
**Pull request by murilovmachado**
## Description
3 new workflows:
- Deploy Webchat to all Staging Helplines
- Deploy Webchat to all Production Helplines
- Deploy Flex to all Production Helplines

Added forms definitions for helpline F2 (which we should reevaluate if we should delete this account).

~TODO: Turns out there's a lot of repeated github code in here. I'll make it more DRY~

It uses a similar system for DRYing workflows as `serverless` (with small differences)

### Related Issues
Fixes [CHI-1756](https://tech-matters.atlassian.net/browse/CHI-1756) and [CHI-1758](https://tech-matters.atlassian.net/browse/CHI-1758)

### Verification steps
After merged, check that both GitHub Actions show up and they work as expected

[CHI-1756]: https://tech-matters.atlassian.net/browse/CHI-1756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHI-1758]: https://tech-matters.atlassian.net/browse/CHI-1758?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: hrm-form-definitions/form-definitions/f2/v1/BlockedEmojis.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[
```
Comment (stephenhand) [inline hrm-form-definitions/form-definitions/f2/v1/BlockedEmojis.json:1]: Was copying these files here a change you intended to push?

File: .github/workflows/webchat-ci.yml [hunk 2]
```diff
@@ -12,118 +12,113 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see https://www.gnu.org/licenses/.
 
-name: Webchat CI
+name: Flex CI
```
Comment (stephenhand) [inline .github/workflows/webchat-ci.yml:15]: Is this naming change correct?

File: .github/workflows/webchat-ci.yml [hunk 3]
```diff
@@ -12,118 +12,113 @@
...
 
 on:
   workflow_call:
 
 jobs:
```
Comment (stephenhand) [inline .github/workflows/webchat-ci.yml:20]: This workflow looks more like it deploys Flex rather than Webchat now?

**Review by stephenhand — COMMENTED** [review]
Couple of nits but just one real question about webchat-ci.yml because it doesn't look right to me but I could be misunderstanding the changes

File: .github/actions/deploy-multiple-helplines/action.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see https://www.gnu.org/licenses/.
+
+# Auxiliary Github Action to deploy a flex or webchat to multiple helplines at once
+name: Deploy Multiple Helplines
```
[the remark this answers is not in the document]
Comment (murilovmachado) [reply .github/actions/deploy-multiple-helplines/action.yml:16]: I confess I tried making it an action, but couldn't get it to work easily. The dev experience and error feedback is so bad, I've just given up and made it a workflow for now.

File: hrm-form-definitions/form-definitions/f2/v1/BlockedEmojis.json (hunk 1 above)
Comment (murilovmachado) [reply hrm-form-definitions/form-definitions/f2/v1/BlockedEmojis.json:1]: Yes, now that we fetch automatically all staging helplines from the Parameter Store, F2 was one of the target helplines. This exposed the issue that we have a helpline without form-definitions in place.

For quick fixing this, I've copied `AS` definition, but left a comment on the PR to re-evaluate if we should actually delete this account.

File: .github/workflows/webchat-ci.yml (hunk 3 above)
Comment (murilovmachado) [reply .github/workflows/webchat-ci.yml:20]: I must have done something wrong during the last commits, I'll fix it. Good catch

File: .github/workflows/webchat-ci.yml (hunk 2 above)
Comment (murilovmachado) [reply .github/workflows/webchat-ci.yml]: Reverted this change, thanks for the catch

**Review by stephenhand — APPROVED** [review]
This is great Murilo, thx!

**Pull request merged**